README.md

```diff
@@ -7,66 +7,47 @@
+:::
 
-### Rule Namespacing
-
-The namespacing format for contributing rules is `<language>.<framework>.<category>.$MORE`. If a `framework` isn't applicable, use `lang` instead.
-
-`category` is one of:
-- security
-- correctness
-- best-practice
-- maintainability
-- performance
+You can also contact us at support@r2c.dev to make Semgrep rule contributions. We will import your rules for everyone to use!
 
-If a `security` rule is discouraging the use of a bad pattern (such as formatted SQL strings), we recommend appending `audit` to your namespace. This distinguishes it from a `security` rule that is specifically aiming to detect a vulnerability.
+## Additional information
 
-<p align="center">
-    <img src="https://web-assets.r2c.dev/semgrep-live-namespacing.png" alt="semgrep.live rule namespace" width="500" />
-</p>
+### Help
 
-### Github Action To Run Tests
+Join [Slack](https://r2c.dev/slack) for the fastest answers to your questions! Or contact the team at support@r2c.dev.
 
-If you fork this repo or create your own, you can add a special [semgrep-rules-test](https://github.com/marketplace/actions/semgrep-rules-test) Github Action to your workflow that will automatically test your rules by running `make test` using the latest version of semgrep.
+### GitHub action to run tests
 
-See ours [here](.github/workflows/semgrep-rules-test.yml)
+If you fork this repository or create your own, you can add a special [semgrep
+-rules-test](https://github.com/marketplace/actions/semgrep-rules-test) GitHub Action to your workflow that will automatically test your rules using the latest version of Semgrep. See our [semgrep-rules-test](https://github.com/returntocorp/semgrep-rules/blob/develop/.github/workflows/semgrep-rules-test.yml).
 
 ### Benchmarks
+
 The [benchmark job](https://github.com/returntocorp/semgrep-rules/actions?query=workflow%3Arule-benchmarks) runs every weekend. It uploads a few artifacts, which can be downloaded. If you download the test logs, there are two relevant pieces of information in there: the benchmark table, which roughly shows the performance of every rule that completes in under 60 seconds, and any failed tests are rules that did not complete within 60 seconds.
 
-To run benchmark tests locally, do the following from the root of `semgrep-rules`:
-```
+To run benchmark tests locally, in your command line, use the following commands in the root directory of `semgrep-rules` local repository clone.
+```bash
 pipenv shell
 pipenv install --dev
 export PYTHONPATH=.
 pytest --timeout=60 --rule-directory=[path_to_rule_directory] --git-repo=[git_URL] tests/performance/test_public_repos.py
 ```
+
 If you omit `--git-repo` from the pytest command, it will run the provided benchmark repo.
+
+### Rulesets
+
+Rulesets are groups of rules organized by purpose (not by language or framework) from Semgrep Registry. If you want to modify existing rulesets or create your own, please contact us at support@r2c.dev.
```

Comment:
Great catch! How's something like:

Rulesets are groups of rules organized by purpose, language, or framework sourced from the Semgrep Registry.

Reply:
Please update this before merging.

Reply:
Thank you! will use Milan's suggestion.

Reply:
I updated the rulesets description and added a link to Contributing docs again in the introduction, so this link is mentioned **twice** now in this document (I am afraid that people could miss it otherwise). Please, have a look at this commit: https://github.com/returntocorp/semgrep-rules/pull/2151/commits/6d9df00c090d0ea8c84dda71de2b9e277ddeaca1 @milanwilliams @s-santillan

Reply:
LGTM!